src/catalog/src/local/memory.rs

```diff
@@ -332,9 +341,19 @@ impl MemoryCatalogManager {
         Ok(true)
     }
 
+    fn create_catalog_entry(self: Arc<Self>, catalog: String) -> SchemaEntries {
+        let information_schema = InformationSchemaProvider::build(
+            catalog.clone(),
```

Comment:
Why clippy doesn't complain about unnecessary `clone`...

Reply:
good eyes... It took me two times to realize 🫣

>Why clippy doesn't complain about unnecessary clones...

I guess the other `catalog` makes things different. Let me check